src/System.Runtime.Numerics/src/System/Numerics/BigIntegerCalculator.Buffer.cs

```diff
@@ -0,0 +1,147 @@
+﻿using System.Diagnostics;
```

Comment:
This project already uses the <outer_type>.<inner_type>.cs conventions, so I would stick to it. And in general, I like the convention, though I am not sure if it's standard over the whole CoreFx repo. @terrajobst, do we have a standard convention for naming files with partial types containing nested type?